Utilities/bootstrap

```diff
@@ -46,6 +47,27 @@ import subprocess
 import sys
 import tempfile
 
+kTargetTriple = {
+    'Darwin': {
+        'x86_64': 'x86_64-apple-macosx10.10',
+    },
+    'Linux': {
+        'aarch64': 'aarch64-unknown-linux-gnu',
+        # TODO(compnerd) support soft float
```

Comment:
We use `FIXME: ...` format in SwiftPM. You can put a tag like "(win)" if you'd like